src/pages/sighting/statusCard/StatusCard.jsx

```diff
@@ -159,15 +167,31 @@ export default function StatusCard({ sightingData }) {
           notStartedText={intl.formatMessage({
             id: 'WAITING_ELLIPSES',
           })}
-          inProgressText={getProgressText(intl, curationStartTime)}
+          inProgressText={
+            hasEditPermission
+              ? intl.formatMessage({ id: 'CURATION_INSTRUCTIONS' })
+              : getProgressText(intl, curationStartTime)
+          }
           finishedText={`Curation finished on ${getDateString(
             curationEndTime,
           )}.`}
           skippedText={intl.formatMessage({
             id: 'CURATION_SKIPPED_MESSAGE',
           })}
           failedText={intl.formatMessage({ id: 'CURATION_FAILED' })}
-        />
+        >
+          {isCurationInProgress && hasEditPermission && (
+            <div style={{ marginTop: 4, marginBottom: 20 }}>
+              <ButtonLink
+                id="CURATION_ASSIGN_ANNOTATIONS"
+                href="#individuals"
+                display="panel"
```

Comment:
Ooo... I dunno. It does seem strange that they are not consistent. But "View Match Results" doesn't seem as urgent because it's not holding up the workflow?

I'm no longer sure what to think. What do _you_ think? 

I think that the primary button itself looks fine. There. It definitely gets my attention.

Reply:
I agree with you that they can be different because of their importance to the IA pipeline, and that the curation buttons should be "primary". If it becomes an issue, I think that "View Match Results" should be changed to "primary" as well, rather than reverting the curation buttons to "panel"

Reply:
TBH, I'm ok with how it is currently: "Assign Annotations" as primary, and "View match results" as panel.